Lib/fontTools/misc/symfont.py

```diff
@@ -101,18 +101,21 @@ def _curveToOne(self, p1, p2, p3):
 # Do not use this in real code.
 # Use fontTools.pens.momentsPen.MomentsPen instead.
 AreaPen = partial(GreenPen, func=1)
-MomentXPen = partial(GreenPen, func=x)
-MomentYPen = partial(GreenPen, func=y)
-MomentXXPen = partial(GreenPen, func=x*x)
-MomentYYPen = partial(GreenPen, func=y*y)
-MomentXYPen = partial(GreenPen, func=x*y)
+Planar1stMomentWrtXPen = partial(GreenPen, func=y)
+Planar1stMomentWrtYPen = partial(GreenPen, func=x)
+Planar2ndMomentWrtXPen = partial(GreenPen, func=y*y)
+Planar2ndMomentWrtYPen = partial(GreenPen, func=x*x)
+ProductMomentXYPen = partial(GreenPen, func=x*y)
 
 
 def printGreenPen(penName, funcs, file=sys.stdout):
 
 	print(
 '''from fontTools.pens.basePen import BasePen
 
+class %sOpenContourError(NotImplementedError):
```

Comment:
this `%s` formatting characters should not be here

Reply:
This whole class we decided should not be here since I moved it to BasePen.